### PR TITLE
cmd/bench: add -p/--threads options to run benchmarks in parallel

### LINKED ISSUE #357: Run benchmark in parallel
`juicefs bench` run basic benchmark using single threads, which can be run in parallel to know  how it works under concurrent workload.

### PR DESCRIPTION
**Pull request by Sandy4999**
close #357

### CODE REVIEW

**Comment by chnliyong** [comment]
Is `-p`(parallel) for short more suitable(`-t` may intuitively means test). In addition, `sync` sub command also use `-p` for short (`--threads` for long option), keep the option name consistent.

File: cmd/benchmark.go [hunk 1]
```diff
@@ -330,6 +320,12 @@ func benchmarkFlags() *cli.Command {
 				Value: 100,
 				Usage: "number of small files",
 			},
+			&cli.IntFlag{
+				Name:    "threads",
+				Aliases: []string{"p"},
+				Value:   3,
```
Comment (davies) [inline cmd/benchmark.go:326]: The default should be 1, to be backwards compatible.

File: cmd/benchmark.go [hunk 2]
```diff
@@ -37,127 +37,125 @@ type Benchmark struct {
...
-	}
-	defer file.Close()
-	for i := 0; i < blockCount; i++ {
-		start := time.Now()
-		if _, err = file.Write(buf); err != nil {
-			return nil, err
-		}
-		if i == blockCount-1 {
-			file.Close()
+func writeFiles(prefix string, fCount, bSize, bCount int, cost chan<- bool) {
```
Comment (davies) [inline cmd/benchmark.go:43]: cost -> progress

**Review by davies — CHANGES_REQUESTED** [review]

File: cmd/benchmark.go (hunk 1 above)
Comment (Sandy4999) [reply cmd/benchmark.go]: ebf2379

File: cmd/benchmark.go (hunk 2 above)
Comment (Sandy4999) [reply cmd/benchmark.go]: ebf2379

**Review by davies — APPROVED** [review]

**Pull request merged**